features/step_definitions/town_steps.rb

```diff
@@ -20,9 +20,15 @@
 end
 
 
+Given(/^my non-feature branches are configured as (.*)$/) do |data|
+  set_configuration 'non-feature-branch-names',
+                    Kappamaki.from_sentence(data).join(', ')
+end
+
+
 Given(/^my perennial branches are configured as (.*)$/) do |data|
-  perennial_branches = Kappamaki.from_sentence(data).join(', ')
-  set_configuration 'perennial-branch-names', perennial_branches
+  set_configuration 'perennial-branch-names',
+                    Kappamaki.from_sentence(data).join(', ')
```

Comment:
The old way needs that smelly and redundant temporary variable. Which (a) has a wrong name, since it contains the names of the perennial branches, and not the branches themselves. And (b) reads pretty redundant: `set_configuration 'perennial-branch-names', perennial_branch_names`.

I find this version more readable and concise. We need this way of formatting anyways, when using methods that take so many arguments that they don't fit on a single line. So using it here seems reasonable.